microceph/cmd/microceph/status.go

```diff
@@ -27,7 +27,7 @@ func (c *cmdStatus) Command() *cobra.Command {
 }
 
 func (c *cmdStatus) Run(cmd *cobra.Command, args []string) error {
-	m, err := microcluster.App(microcluster.Args{StateDir: c.common.FlagStateDir, Verbose: c.common.FlagLogVerbose, Debug: c.common.FlagLogDebug})
```

Comment:
Hmmmmm this is a little confusing. Can this change be described a little (dropping the Verbose and Debug flags.)

Reply:
These args only applied to the daemon as it starts, but since we use the same struct to set up the microcluster client, these fields were populated even though they are never read. 

We have split the daemon and client args now, so these unused fields were removed.

Reply:
The flags are defined here now:
https://github.com/canonical/microceph/blob/fd6a044aa00a5201237c339755f8b4973bfeb4ae/microceph/cmd/microcephd/main.go#L90

This is the same place that they were read from before, but now the args type is different.

Reply:
Ack thanks for this :D